x/bep3/client/cli/query.go

```diff
@@ -293,8 +327,14 @@ $ kvcli q bep3 swaps --page=2 --limit=100
 				return fmt.Errorf("No matching atomic swaps found")
 			}
 
+			formattedAtomicSwaps := []FormattedAtomicSwap{}
+
+			for _, swap := range matchingAtomicSwaps {
+				formattedAtomicSwaps = append(formattedAtomicSwaps, NewFormattedAtomicSwap(swap))
+			}
+
 			cliCtx = cliCtx.WithHeight(height)
-			return cliCtx.PrintOutput(matchingAtomicSwaps.String()) // nolint:errcheck
+			return cliCtx.PrintOutput(formattedAtomicSwaps)
```

Comment:
I just matched the existing output of the CLI command, but it doesn't have parity with the REST API.

Are we OK with that?  Should we just return the existing AtomicSwap struct in the cli?

The biggest downside might be the `ID` isn't included and the random number hash is a yaml array instead of a hex string.

Reply:
Makes sense, thanks for sharing thoughts on this.  I'm going to update the command and rest api to match.